packages/forms-web-app/src/views/pdf/source.njk

```diff
@@ -0,0 +1,319 @@
+{% extends "../layouts/main.njk" %}
```

Comment:
actual lift+shift of the check-your-answers page so i had a thing that should render some data that we could hack back to what we needed.. may not be the most progressive route..